---
title: YILEI FU's Homepage
tags: 
    - curriculum vitae
date: 2018-09-01 05:23:13
---
**Now looking for Ph.D. opportunity.**
>**Address: Harbin Institute of Technology** <a href = "http://en.hit.edu.cn/"><small>(HIT)</small></a> 92 West Dazhi Street, Nan Gang District, <a  href = "https://www.google.com/maps/place/%E5%93%88%E5%B0%94%E6%BB%A8%E5%B7%A5%E4%B8%9A%E5%A4%A7%E5%AD%A6/@45.7470436,126.6312419,17z/data=!3m1!4b1!4m5!3m4!1s0x5e447f9d2dd87cd1:0xcb3b9952a10fed6c!8m2!3d45.7470436!4d126.6334305">Harbin, Heilongjiang, China</a>
> **E-mail:** fuyilei96 AT hit DOT edu DOT cn
> <a href="https://github.con/Fu-Yilei"><strong>Visit My GitHub Page</strong></a>

### Yilei Fu's Research Statement <small> as an undergraduate student</small>
<p align="justify">
    During my undergraduate study procedure, I have accessed lots of different areas in computer science. I took many courses in diverse research fields, such as machine learning and deep learning, computational bioinformatics, computer vision, and natural language processing. I joined the massive data computing lab in our university for my own interest because I believe massive data research experience can be a good foundation for expanding my future research career in different research fields. In the lab, I was supervised by Prof. Hongzhi Wang for a topic of dealing with massive queries on knowledge bases when people executing data cleansing tasks on it. I produced a novel efficient structure for combining queries in graph form, and do the graph matching tasks on RDF formed knowledge bases. With the foundation of the previous research experience, I have already known the basic procedure of doing research. With the more I learn in the university, the more I think I should explore in unknown fields. There are many fields that I am currently interested to, such as bioinformation, NLP, and machine learning. In my opinion, today’s scientific research is more and more rely on massive data, like gene data, vision data and text data, etc. Biocomputing and NLP are two major paths I would like to explore more in the future, but I might only be able to do either one. In the Biocomputing field, I have joined HIT's Bioconputing lab for one year, and published two papers about DNA sequenceing. As to Natrual Language Processing, I am mainly interested in information retrival and text data processing. All these work I mentioned above requires massive data computing techniques and artificial intelligence methods, which I have already explored boardly during my undergraduate study procedure.<br>
    <strong>My research interests</strong> can be briefly described by solving different problems through using massive and artificial intelligence methods, which includes bioinformatics, natural language processing and machine learning algorithms developing.
</p>
<br>

### Education Backgrounds
+ <strong>2014.09.01-2018.06.31:</strong>
 Harbin Institute of Technology (HIT), B.E. major in computer science and technology.
+ <strong>2018.01.01-now:</strong>
 Harbin Institute of Technology (HIT), HIT Bioinformatics Centre. </li>
+ <strong>Scores:</strong>
    - Major Weighted Average Score: 90.41/100 (4.0 in scale of 4.0)
    - TOEFL: 101 (R25 S23 L26 W27)
    - GRE: 316 (V147 Q169)
  
### Research Experiences
+ <strong>2015</strong> Data processing work in Institute of Extreme Environmental Nutrition and Protection at HIT.
+ <strong>2016.9-</strong> Research in Massive Data Computing Lab at HIT.
+ <strong>2018.01.01-now</strong> Harbin Institute of Technology (HIT), HIT Bioinformatics Center. 

<strong>Publications and papers working on:</strong>
+ <a href="https://www.sciencedirect.com/science/article/pii/S0753332217334807#!">Acanthopanax Senticosus Reduces Brain Injury in Mice Exposed to Low LET Radiation</a> (Yingyu Zhou, Wei Song, <strong>Yieli Fu</strong>, Denis Baranenko, Jiaping Wang, Yongzhi Li and Weihong Lu.) In BIOPHA'2017
    + Irradiated mice administered AS extract showed improved learning ability and central nervous system (CNS) function. AS extract effectively controlled nerve cell swelling, protein loss, and necrotic tissue liquefaction that was observed in the irradiated mouse brain. Metabolomics data demonstrated that treatment with AS extract resulted in significant quantitative changes of 16 classes of cerebral metabolites in the prefrontal cortex (PFC) of irradiated mice. Using a principal component analysis (PCA), three principal components, F1, F2 and F3, were identified as related to brain energy metabolism, brain tissue development, and brain glutamate cycle, respectively. In addition, the F2 and F3 scores of the AS-treated group of mice were higher compared to mice that were treated with Venlafaxine. Furthermore, the efficiency of balancing the glutamate cycle that the AS-treated group achieved was two times greater than that of the mice treated with Venlafaxine.
+ deGSM: memory scalable construction of large scale de Bruijn Graph (Hongzhe Guo, <strong>Yilei Fu</strong>, Yan Gao, Junyi Li, Yadong Wang and Bo Liu) In APBC'2019. Yilei Fu and Hongzhe Guo have same contributions to this paper.
    + We propose a lightweight parallel de Bruijn graph construction approach, de Bruijn Graph Constructor in Scalable Memory (deGSM). The main idea of deGSM is to efficiently construct the Burrows-Wheeler Transformation (BWT) of the unipaths of de Bruijn graph in constant RAM space and transform the BWT into the original unitigs. It is mainly implemented by a fast parallel external sorting of k-mers, which allows only a part of k-mers kept in RAM by a novel organization of the k-mers. The experimental results demonstrate that, just with a commonly used machine, deGSM is able to handle very large genome sequence(s), e.g., the contigs (305 Gbp) and scaffolds (1.1 Tbp) recorded in GenBank database and Picea abies HTS dataset (9.7 Tbp). Moreover, deGSM also has faster or comparable construction speed compared with state-of-the-art approaches. With its high scalability and efficiency, deGSM has enormous potentials in many large-scale genomics studies.
+ rCANID: read Clustering and Assembly-based Novel Insertion Detection tool (<strong>Yilei Fu</strong>, Tao Jiang, Bo Liu and Yadong Wang) In BIBM'2018.
    + We propose Read Clustering and Assembly-based Novel Insertion Detection tool (rCANID), a novel long read-based NSI detection approach. rCANID fully takes the advantage of chimerically aligned and unaligned reads by its speciﬁcally designed read clustering and light weight local read assembly methods to effectively reconstruct inserted sequences with relatively low computational cost. Benchmarking on both of simulated and real datasets demonstrates that rCANID can sensitively discover NSIs, especially for those having large inserted novel sequences, which could be hard to state-of-the-art approaches. rCANID is suited to be integrated into many computational pipelines to play important roles in many genomic studies.
+ Fast variation-aware read alignment with deBGA-VARA (Hongzhe Guo, Bo Liu, Dengfeng Guan, Yilei Fu, Yadong Wang) In BIBM’2018.
    + We propose a variation-aware read alignment algorithm (VARA), which generates the alignment between read and multiple genomic sequences simultaneously utilizing the schema of the Landau-Vishkin algorithm. VARA dynamically extracts regional variants to construct a pseudo tree-based structure on-the-ﬂy for seed extension without loading the whole genome variation into memory space. We developed the novel high throughput sequencing read aligner deBGA-VARA by integrating VARA into deBGA. DeBGA-VARA is benchmarked both on simulated reads and the NA12878 sequencing dataset. The experimental results demonstrate that read alignment incorporating genetic variation knowledge can achieve high sensitivity and accuracy. Moreover, due to its efﬁciency, VARA provides a promising solution for further improvement of variant calling while maintaining small memory footprints. DeBGA-VARA is available at: https://github.com/hitbc/deBGAVARA
+ Multi-Query Processing on Knowledge Based Data Cleansing (<strong>Yilei Fu</strong>, Hongzhi Wang)
    + We developed a multi-query tasks combination strategy to improve query eﬃciency on massive data. Our algorithms treated SPARQL queries as graphs, then combined them into a new hypergraph. In addition, our algorithms can also extract queries from combined hypergraphs. After bring up our algorithms, we used cost model to analyze our algorithms and developed some combination strategies. 
### Awards and Honors
+ <strong>2014.12</strong> Special Prize People's Scholarship
+ <strong>2015-2016</strong> 2nd Prize People's Scholarship*4
+ <strong>2015-2016</strong> MCM/ICM Honorable Mention (Team Leader)
+ <strong>2016.10</strong> National Computer Simulation Grand Prix 2nd Prize
+ <strong>2016.11</strong> Leading Talented People Project at Harbin Institute of Technology
### Main Experience
+ <strong>2014.11-2015.07</strong> Leader of Undergraduate Annual Project
    
    <small>Led a group of four finished an Application on Android that can gather information and show high frequency words, instructed by Prof. Wanxiang Che, Information Retrieval Center of Harbin Institute of Technology.</small>
+ <strong>2014.08-2014.08</strong> Harbin Aircraft Industry Group Co. Ltd.
    
    <small>Got a preliminary understanding of computer, industrial control system management and its basic principles.</small>
+ <strong>2014.09-2015.09</strong> Class Monitor
+ <strong>2014.9-2015.3</strong> Institute of Extreme Environmental Nutrition and Protection
    
    <small>Dominant developer of Cold Region Extreme Environment Functional Components Mining Database.</small>
+ <strong>2016.7-2016.7</strong> TA at CS Department of HIT
    
    <small>Helped Prof. Asim Smailagic and Dan Siewiorek (Two Professors in CMU, CS Department)’s translation work and their courses when they were visiting at HIT’s CS Department.</small>
+ <strong>2016.7-2016.8</strong> Giant KONE Elevator Co. Ltd.
    
    <small>Translate literatures about the technical documents and new equipment description. Explaining the principles of the machine to the technical workers.</small>
+ <strong>2017.1-2017.6</strong> Concurrent Enrollment Program at UC Berkeley
    
    <small> Selected to attend the extension program at UC Berkeley, and got enrolled in several courses (Data Science, Computer Network, Database Management and Website Design). Average GPA: 3.33.</small>
+ <strong>2014- </strong> 500+ Hours Volunteer in Disabled Activity Center
    
    <small> Helped Children with mental handicap with their daily activities, and helped with the environment protecting work of the activity center. </small>
+ <strong>2018.4.12-14 </strong> <a href="https://tl.hku.hk/c91-2018/">Group Leader, Poster Designer and Main Repensenter of Innovation and Excellence in Teaching and Learning in Research Intensive Universities in China</a>
    
    <small>Team leader of the conference “Innovation and Excellence in Teaching and Learning in Research Intensive Universities in China”, hosted by Hong Kong University. Domainated poster design and being the main repersenter of student sharing session. Only 6 undergraduate students in my Harbin Institute of Technology were invited by this conference. </small>


### Beyound My Academic Life
+ Lead Guitarist, supplementary Drummer of a Diversified Rock Band “Zale” at Harbin Institute of Technology.
+ Music Producer. You can hear my producition <a href="https://music.163.com/#/djradio?id=349481065">here</a>.
+ There are more than 4 months temperature is below 0 degree here in <a href="https://en.wikipedia.org/wiki/Harbin">Harbin</a>, so I am a good ice skater.
